docs/install.rst

```diff
@@ -70,6 +70,10 @@ need to install Python 3 with virtualenv in your system as well.
 
         sudo apt-get install redis-server
 
+.. note::
+
+   Some unit tests depend on Hg (mercurial).
+   Be sure that Hg is `installed <https://www.mercurial-scm.org/wiki/Download>`_ if you want to see all the tests pass.
```

Comment:
Similar to other PRs, i'd like to instead fix tests with mercurial if possible. Could this be rebased out?